statistics/semantic/src/main/java/com/spotify/heroic/statistics/semantic/SemanticConsumerReporter.java

```diff
@@ -40,6 +41,8 @@
     private final Meter messageIn;
     private final Meter messageError;
     private final Meter consumerSchemaError;
+    private Optional<Long> consumerThreadsWanted;
```

Comment:
This needs to be synchronized, like an `AtomicLong`. Otherwise there are no guarantees updates will be visible across threads.